vscode/tsconfig.json

```diff
@@ -2,15 +2,15 @@
   "extends": "../tsconfig.json",
   "compilerOptions": {
     "module": "commonjs",
-    "target": "ES2023",
+    "target": "esnext",
     "baseUrl": ".",
     "outDir": "out",
     "lib": ["ES2020"],
     "sourceMap": true,
     "rootDir": "src",
     "strict": true,
-    "composite": true
+    "composite": true,
+    "declaration": true
   },
-  "include": ["src/**/*.ts"],
-  "references": [{ "path": "../shared" }]
```

Comment:
the config worked for me without references but it puzzles me - is it because we only have types in `shared`?

Reply:
I agree the references need a second look & perhaps a TODO commend. The way I understand it - since we build the shared package before running dev mode ( or any build / dev of the webview or vscode packages), we should have everything available as the shared package is imported in the package.json for those respective packages